frontend/src/components/Plan/PlanIdeaList.tsx

```diff
@@ -0,0 +1,22 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { icon } from '@fortawesome/fontawesome-svg-core/import.macro'
+import styles from './PlanIdeaList.module.css'
+
+export const PlanIdeaList = (): JSX.Element => {
+  return (
+    <div className={styles.ideaList}>
+      <div className={styles.ideaListBlock}>
+      <h4 id='idea-list-heading'>Idea list</h4>
```

Comment:
any reason this needs to be an id, rather than just a class?

Reply:
i probably put ID here as a placeholder, will come back to this later when I figure out the idea list component